src/dlangui/widgets/layouts.d

```diff
@@ -519,10 +519,14 @@ class ResizerWidget : Widget {
     private void resizeAndFireEvent(short newWidth, ResizerEventType type)
     {
         // Respect the dimensions
-        if(newWidth > minPreviousItemWidth && newWidth < (parent.width - minWidth - minNextItemWidth) &&
-           newWidth > _previousWidget.minWidth() && newWidth < (parent.width - minWidth - _nextWidget.minWidth()))
+        if( (newWidth > minPreviousItemWidth) && (newWidth < (parent.width - minWidth - minNextItemWidth)) && (_previousWidget is null ||
```

Comment:
Is it intended to be `_previousWidget is null` ? Appears like you meant `!is null` instead

Reply:
_previousWidget can be null -- at least true for dmledit resizing.

Reply:
Oh yea, I see what you did there now!